.github/workflows/e2e_tests.yml

```diff
@@ -30,11 +30,11 @@ jobs:
         id: run-azure-e2e-tests
         run: ./gradlew e2eTest --tests AzureE2eTest
         env:
-          # These values were copied from jvm-lab/identity-federation stack's output
-          # The Azure tenant/subscription that is used here belongs to @myhau (Michal Fudala)
-          ARM_TENANT_ID: '1ff3e4bd-41d1-4df2-b776-8d3fcbf77184' # tenant-id output of jvm-lab/identity-federation/dev stack
-          ARM_SUBSCRIPTION_ID: '3565bcd1-dd9b-45a8-9d7c-8880aaaa9a9f' # subscription-id output of jvm-lab/identity-federation/dev stack
-          ARM_CLIENT_ID: '4169b156-839a-48c2-846f-d7c079331c71' # service-principal-client-id output of jvm-lab/identity-federation/dev stack
+          # These values were copied from jvm-lab/identity-federation/dev stack's output
+          # The Azure tenant/subscription that is used here belongs to @jplewa (Julia Plewa)
```

Comment:
We are using VirtusLab's tenant/subscription (not yours), so this comment should reflect that.

Reply:
True haha. I didn't really read the comment, I just instinctively replaced your username with mine 🙈